app/models/training_module.rb

```diff
@@ -37,7 +37,7 @@ def self.trim_id_from_filename
   end
 
   def self.load(slug: nil)
-    TrainingBase.load(content_class: self, slug_list: slug)
+    TrainingBase.load(content_class: self, slug_list: slug.nil? ? nil : [slug])
```

Comment:
Is there anywhere that calls `load` without the argument, or can we remove the default and just always pass `[slug]`?

Reply:
The `load` with arguments is used only in `reload_module`. Otherwise it is just `.load`